src/components/LabCard/LabCard.js

```diff
@@ -0,0 +1,218 @@
+    if (lowVal) return (`>= ${lowVal.value} ${lowVal.unit || ''}`);
+    if (highVal) return (`<= ${highVal.value} ${highVal.unit || ''}`);
+    return '';
+  }
+
+  const HealthCardLabResult = ({ labResult }) => (
+    <Box className={[styles.flexCol, styles.labRoot].join(' ')}>
+      <Box className={styles.divider}>
+        <Divider className={styles.line2} />
+      </Box>
+      <Grid container className={styles.grid}>
+        <Grid container item className={styles.gridRow} spacing={0}>
+          <Grid item xs={3} sm={2} className={styles.gridLabel}>
+            <Typography>{t('healthcarddisplay.Lab Test')}</Typography>
+          </Grid>
+          <Grid item xs={9} sm={10} className={styles.gridItem}>
+            <Typography>
+              <Box component="span" fontWeight="700">
+                {displayLabTestCode(labResult.code)}
+              </Box>
+            </Typography>
+          </Grid>
+        </Grid>
+
+        <Grid container item className={styles.gridRow} spacing={0}>
+          <Grid item xs={3} sm={2} className={styles.gridLabel}>
+            <Typography>{t('healthcarddisplay.Date')}</Typography>
+          </Grid>
+          <Grid item xs={9} sm={10} className={styles.gridItem}>
+            <Typography className={styles.date}>
+              {displayDateTime(labResult.effectiveDateTime)}
+            </Typography>
+          </Grid>
+        </Grid>
+
+        <Grid container item className={styles.gridRow} spacing={0}>
+          <Grid item xs={3} sm={2} className={styles.gridLabel}>
+            <Typography>{t('healthcarddisplay.Lab Performer')}</Typography>
+          </Grid>
+          <Grid item xs={9} sm={10} className={styles.gridItem}>
+            {labResult.performer && labResult.performer.length > 0 && (
+              <Typography>{labResult.performer[0].display}</Typography>
+            )}
+          </Grid>
+        </Grid>
+
+        <Grid container item className={styles.gridRow} spacing={0}>
+          <Grid item xs={3} sm={2} className={styles.gridLabel}>
+            <Typography>{t('healthcarddisplay.Test Result')}</Typography>
+          </Grid>
+          <Grid item xs={9} sm={10} className={styles.gridItem}>
+            <Typography>
+              {displayLabTestResult(labResult)}
+            </Typography>
+          </Grid>
+        </Grid>
+
+        {labResult.referenceRange && labResult.referenceRange.length > 0 && (
+          <Grid container item className={styles.gridRow} spacing={0}>
+            <Grid item xs={3} sm={2} className={styles.gridLabel}>
+              <Typography>{t('healthcarddisplay.Reference Range')}</Typography>
+            </Grid>
+            <Grid item xs={9} sm={10} className={styles.gridItem}>
+              <Typography>{displayReferenceRange(labResult.referenceRange[0])}</Typography>
+            </Grid>
+          </Grid>
+        )}
+
+      </Grid>
+    </Box>
+  );
+
+  return (
+    <Box display="flex" className={styles.healthCard}>
```

Comment:
This wrapping portion and styling is shared between lab and immunizations and should be consolidated in a base component.

Reply:
Good call, I agree. I made a `HealthCard` component which returns the Card itself. This contains the patient information, and Issuer. It wraps either the `VaccineCard` or the `LabCard` depending on which `credentialType` is presented.